Modelica/Magnetic/FluxTubes/Examples/BasicExamples/ToroidalCoreAirgap.mo

```diff
@@ -97,8 +97,8 @@ Using the values shown in section Parameters, the results can be validated easil
 </p>
 <table border=\"1\" cellspacing=\"0\" cellpadding=\"2\">
 <tr><th>element   </th><th>cross section     </th><th>length             </th><th>rel. permeability </th><th>B                   </th><th>H                                    </th><th>mmf              </th></tr>
-<tr><td>core      </td><td>d<sup>2</sup>*pi/4</td><td>r*alfa             </td><td>&mu;<sub>r</sub>  </td><td>flux / cross section</td><td>B/(&mu;<sub>r</sub>*&mu;<sub>0</sub>)</td><td>H*length         </td></tr>
-<tr><td>airgap    </td><td>d<sup>2</sup>*pi/4</td><td>delta=r*(2*pi-alfa)</td><td>1</td><td>flux / cross section</td><td>B/(&mu;<sub>0</sub>)</td><td>H*delta         </td></tr>
+<tr><td>core      </td><td>d<sup>2</sup>*pi/4</td><td>r*alpha             </td><td>&mu;<sub>r</sub>  </td><td>flux / cross section</td><td>B/(&mu;<sub>r</sub>*&mu;<sub>0</sub>)</td><td>H*length         </td></tr>
+<tr><td>airgap    </td><td>d<sup>2</sup>*pi/4</td><td>delta=r*(2*pi-alpha)</td><td>1</td><td>flux / cross section</td><td>B/(&mu;<sub>0</sub>)</td><td>H*delta         </td></tr>
```

Comment:
`delta=r*(2*pi-alpha)`

should be enhanced to either

`<code>delta=r*(2*pi - alpha)</code>` or `&delta;=<code>r</code>*(2*&pi; - &alpha;)`

Reply:
To be consistent, shall we add `<code>` to all the variables of the tables, except for the Greek variables? If yes, such change affects a lot of other models as well...

Reply:
`<code>` should work even for the greek once.

Reply:
@christiankral, @beutlich although the `<code>` would be nice to have I don't think they need to be implemented right now. They can always be added at a later point. I suggest to open up an enhancement ticket for that and merge this PR as is for now.

Reply:
OK. Please go ahead.

Reply:
Yes, I agree. Let's open another ticket for `<code>` and merge this PR.